torch/_inductor/kernel/mm.py

```diff
@@ -180,7 +180,9 @@ def tuned_mm(mat1, mat2, *, layout=None):
                 **mm_options(config, m, n, k, layout),
             )
     if static_shape and is_nonzero and use_cutlass_template(layout, m, n, k):
-        CUTLASS3xGemmTemplate.add_cutlass_gemm_choices(choices, layout, [mat1, mat2])
+        # Developing on A100 rn.
+        CUTLASS2xGemmTemplate.add_cutlass_gemm_choices(choices, layout, [mat1, mat2])
+        # CUTLASS3xGemmTemplate.add_cutlass_gemm_choices(choices, layout, [mat1, mat2])
```

Comment:
Need to figure out how to know when to use v2 vs v3.

Reply:
Perhaps we could stick with the latest version, e.g. v3?

Reply:
In general don't use v2. There's no performance benefit over Triton. I originally entirely removed Cutlass v2 support, but it was re-added since an external contribution required it for certain mixed-mode GEMMs that were using 2.x APIs.

Reply:
Thanks for the context Kai!